packages/host/src/api/api.ts

```diff
@@ -1,10 +1,27 @@
+import { UpdateSyncSettingsInput } from '@common/types';
 import { Sdk } from './operations.generated';
 
 export const getHostQueries = (sdk: Sdk) => ({
   get: {
-    version: () => async () => {
+    settings: async () => {
+      const result = await sdk.serverSettings();
+      return result.serverSettings;
+    },
+    version: async () => {
       const result = await sdk.apiVersion();
```

Comment:
Think I mentioned it already, probably a good idea to merge the server status, sync progress, api version into a `/status` endpoint...

Reply:
would be neat, yeah. I can see that being very useful